modules/events/events-page/layout/sidebar/total-events/NavGroup.tsx

```diff
@@ -4,7 +4,7 @@ import NavItem from './NavItem'
 type Props = {}
 
 export default function NavGroup() {
-    const totalEventsTitles = ['Visit Page', 'Add to Cart', 'New Order', 'User Identified', 'Login']
+    const totalEventsTitles = ['Visit Page', 'Add to Cart', 'New Order', 'Login']
```

Comment:
Extract the keys of the value of 'selectEventTotals' using Object.keys() instead of using a static list to avoid undefined values

Reply:
Okay